README.md

```diff
@@ -2,7 +2,91 @@
 
 ![](https://github.com/lendi-au/Ralph/workflows/Build/badge.svg)
 
-Ralph is a CLI tool that automates security incident response on AWS.
+Ralph is a CLI tool that automates security incident response for AWS resources.
 
 It's the tool you use when this happens:
 ![Ralph](./img/ralph.jpg "Ralph")
+
+## Features
+
+Ralph loads and executes a collection of pre-defined runbook to automate incident response.
+
+### EC2 Runbooks
+
+1. Removing IAM Instance Profile
+
+- Detaches the current IAM Instance Profile of an instance.
+
+2. Enable Termination Protection
+
+- Enables termination protection for an instance. This gives an extra step to prevent accidental termination of an instance.
+
+3. Set Shutdown Behavior to Stop
+
+- Setting shutdown behavior to stop guarantees that the shutting down the instance will not result to termination of the instance.
+
+## Installation
+
+To install Ralph using NPM, run:
+
+```lang=bash
+$ npm i @lendi-oss/ralph
```

Comment:
I think we've now changed this to `@lendi/ralph`

Reply:
I tried to run the command. Seems like it' still under @lendi-oss/ralph. I used this as reference - https://www.npmjs.com/search?q=%40lendi%2Fralph.

Reply:
I think it'll get moved to @lendi when we publish using the new NPM token though